docs/cookbook/adding-new-attribute-to-an-entity.md

```diff
@@ -245,9 +245,9 @@ as it is a part of your open-box, eg. [`detail.html.twig`](../../project-base/sr
 ```
 
 ## Data fixtures
-You can modify data fixtures as they are located in ShopBundle.
-### 1. Random `extId`
-If you want to add unique random `extId` every product, you can add the generation to function `createProductDataFromRowForFirstDomain` in [`ProductDataFixtureLoader.php`](../../project-base/src/Shopsys/ShopBundle/DataFixtures/Demo/ProductDataFixtureLoader.php).
+You can modify data fixtures as they are located in project-base.
```

Comment:
```suggestion
You can modify data fixtures in `src/Shopsys/ShopBundle/DataFixtures/` of your project.
```
i am not sure about this one, maybe it should be decided during BV

Reply:
I'll discuss it during BV